packages/design-system/src/components/tooltip/index.js

```diff
@@ -166,11 +166,11 @@ function BaseTooltip({
             anchor: forceAnchorRef || anchorRef,
             popup,
             isRTL,
-            ignoreMaxOffsetY,
+            ignoreMaxOffsetY: true,
```

Comment:
If you explicitly set this to `true`, you won't need to pull it in as a prop any longer, which should be fine for tooltip.  I don't think it gets used anywhere. It was added for `Dropdowns`, which use `Popup`, once we separated `Tooltip` from `Popup` it was no longer need as a prop for `Tooltip`.  The `getOffset` util is still shared between the two which is why it was getting sent down. So long story short, this should be 💯  We had plans to pull separate out `Dropdown` and `Popup` as well, but decided it wasn't really worth it.

Reply:
Okay cool, Thanks for looking into that and for the insight! Ill remove this prop 👍